.github/workflows/release.yml

```diff
@@ -115,7 +113,7 @@ jobs:
         with:
           submodules: recursive
       - name: Install Rust
-        run: rustup update "stable" --no-self-update && rustup default "stable""#
+        run: rustup update "stable" --no-self-update && rustup default "stable"
```

Comment:
we should listen to the cargo-dist warning and remove `rust-toolchain-version` from our Cargo.toml

Reply:
Good call, done.